tests/test_inference.py

```diff
@@ -40,6 +42,35 @@ def dummy_metadata():
     }
 
 
+def test_errors():
+    dummy_target = [0, 1, 0, 1, 0]
+    dummy_middle_output = [1, 1, 0, 1, 0]
+
+    assert hamming_loss(dummy_target, dummy_middle_output) == pytest.approx(
+        0.2, abs=1e-5
+    )
+    assert zero_one_loss_ceil(dummy_target, dummy_middle_output) == pytest.approx(
+        1.0, abs=1e-5
+    )
+
+    dummy_correct_output = [0, 1, 0, 1, 0]
+
+    assert hamming_loss(dummy_target, dummy_correct_output) == pytest.approx(
```

Comment:
we don't need to test sklearn's function so can remove this

Reply:
Also removed this test